tools/hive/ubuntu/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:trusty
+
+USER root
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-oracle
+ENV HIVE_HOME /usr/local/hive
+ENV HADOOP_HOME /usr/local/hadoop
+
+ENV PATH $PATH:$JAVA_HOME/bin:$HIVE_HOME/bin:$HADOOP_HOME:$HADOOP_HOME/bin
+
+# install dev tools
+RUN apt-get update
+RUN apt-get install -y curl wget tar openssh-server openssh-client rsync python-software-properties apt-file apache2
+
+# for running sshd in ubuntu trusty. https://github.com/docker/docker/issues/5704
+RUN mkdir /var/run/sshd
+RUN echo 'root:secretpasswd' | chpasswd
+RUN sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN echo "ServerName localhost" >> /etc/apache2/apache2.conf
+RUN sed -i 's/Listen 80/Listen 9999/g' /etc/apache2/ports.conf
```

Comment:
Is there a reason that we need an Apache webserver too?

Reply:
I don't think this is actually used. I will remove it.